project/roadmap.md

```diff
@@ -4,26 +4,28 @@
 
 If you are interested in collaborating on any of the roadmap features or other features in the project, please mail [Mbed TLS](https://lists.trustedfirmware.org/mailman/listinfo/mbed-tls) or [PSA Crypto](https://lists.trustedfirmware.org/mailman/listinfo/psa-crypto) mailing lists.
 
-**2024 CQ2 (In Development)**
-
-* [Mbed TLS] Mbed TLS4.0 Preparations & Planning
+**2024 CQ3 (In Development)**
+* [Mbed TLS] [Mbed TL3.6.1](<https://github.com/orgs/Mbed-TLS/projects/1#column-19883226>)
+* [PSA Crypto] [PSA Crypto Repo split foundation1](<https://github.com/orgs/Mbed-TLS/projects/1#column-19914196>)
```

Comment:
Can we have something a bit more meaningful than “foundation1” etc?

Reply:
Changed to Preparation similar to the Mbed TLS4.0 items